port/resource_port_blueprint.go

```diff
@@ -358,23 +383,28 @@ func writeBlueprintFieldsToResource(d *schema.ResourceData, b *cli.Blueprint) {
 		} else {
 			p["required"] = false
 		}
+
 		if v.Default != nil {
+			defaultKey := "default_value"
```

Comment:
In case it's the new one, it should be a map of string. It should have a different treatment after the switch case (add the casted string value to a key called "value")